docs/publish/deploy-office-add-in-sso-to-azure.md

```diff
@@ -0,0 +1,282 @@
+1. Select **Windows** for the operating system.
+1. Select the location you want to serve your app from. For example, *West Europe*.
+1. Select the App Service plan you want to use. If you don't have an App Service plan, select **Create new App Service plan**, then enter a name for the plan (such as *AppServiceQS-plan*), then select **F1 Free**.
+1. For **Select an Application Insights resource for your app**, select **Skip for now** and wait the resources to be provisioned in Azure. When prompted to deploy, don't deploy the add-in yet. You'll do that in a later step.
+1. In the **App Service** explorer in Visual Studio code, expand the node for the new app, right-click **Application Settings**, and select **Add New Setting**:
+
+    :::image type="content" source="../images/azure-app-service-add-setting.png" alt-text="Add app setting command.":::
+
+1. Enter `SCM_DO_BUILD_DURING_DEPLOYMENT` for the setting key.
+1. Enter `true` for the setting value.
+
+    This app setting enables build automation at deploy time, which automatically detects the start script and generates the *web.config* with it.
+
+-----
+
+1. Right-click your App Service app and select **Open in Portal**.
+1. When the portal opens in the browser, copy the domain name of the **URL** (not the `https://` part) from the **Overview** pane and save it. You'll need it in later steps.
+
+## Update package.json
+
+1. Open the package.json file. Then replace the `start` command in the `"scripts"` section with the following entry.
+
+    ```json
+    "start": "node middletier.js",
+    ```
+
+1. Find the `"prestart"` entry in the '"scripts"' section and delete it. This section is not needed for this deployment.
+1. Save the file.
+
+## Update webpack.config.js
+
+1. Open the **webpack.config.js** file.
+1. Set the `urlDev` and `urlProd` constants to the following values (without the `https` protocol portion). This will cause webpack to replace `localhost:3000` with your web site domain name in the `/dest/manifest.xml` file.
+
+    ```javascript
+    const urlDev = "localhost:3000";
+    const urlProd = "<your-web-site-domain-name>";   
+    ```
+
+1. Find the first `CopyWebpackPlugin` section and update it to also copy the package.json file to the dist folder as shown in the following example.
+
+    ```javascript
+    new CopyWebpackPlugin({
+        patterns: [
+        {
+            from: "assets/*",
+            to: "assets/[name][ext][query]",
+        },
+        {
+            from: "package.json",
+            to: "package.json",
+        },
+        {
+            from: "manifest*.xml",
+            to: "[name]" + "[ext]",
+            transform(content) {
+                if (dev) {
+                    return content;
+                } else {
+                    return content.toString().replace(new RegExp(urlDev, "g"), urlProd);
+                }
+            },
+        },
+      ],
+    }),
+    ```
+
+1. Save the file.
+
+## Update manifest
```

Comment:
Don't you need to change all references to localhost:3000 in the manifest?

Reply:
You could do it either way. Replace contoso with localhost and the previous step will auto-transform it. Or just follow the steps below and use your actual domain (as contoso isn't yours anyway.) My plan is to follow up with a PR on the template to change this to localhost:3000 and then this step will go away.